python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -552,19 +553,18 @@ def log_cond_tau_post(tau):
 ks = ot.KernelSmoothing()
 posterior = ks.build(post_sample)
 
-# Optional: change the bandwidth
-bandwidth = ks.getBandwidth()
-for k in range(len(bandwidth)):
-    bandwidth[k] = 1.0 * bandwidth[k]
-posterior = ks.build(post_sample, bandwidth)
-
 grid = ot.GridLayout(1, 3)
 grid.setTitle("Bayesian inference")
 xlabs = [r"$\theta_0$", r"$\theta_1$", r"$\tau$"]
+final_value = sample[-10:-1].computeMean()
```

Comment:
If I remember correctly, we decided at the technical committee to represent the true value of the parameters. I am not sure what the point of showing the average of the final values of the MCMC chain is.

Reply:
I thought it would be more robust than the last value to represent the converged value, what should I plot then ?

Reply:
Tagging @MerlinKeller for comment, but to me the final value is not particularly significant for our purposes. Is is but one sample from the posterior distribution.

But since this example uses artifically generated data, you could plot the *true* value of the parameters for comparison.

Reply:
oh yes the true value